pkg/ocm/sca/architectures_gather.go

```diff
@@ -0,0 +1,37 @@
+package sca
+
+import (
+	"context"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Mapping of kubernetes architecture labels to the format used by SCA API
+var kubernetesArchMapping = map[string]string{
```

Comment:
I am not sure if we can use only the map or if we need a function to return a default value when no value is found.

Reply:
Totally agree. Let's add a little function to return the arch from operator's node as default value. That could even help with testing.